micromamba/tests/test_remove.py

```diff
@@ -77,7 +79,9 @@ def test_remove_orphaned(tmp_home, tmp_root_prefix, tmp_xtensor_env, tmp_env_nam
         1 if helpers.dry_run_tests == helpers.DryRun.DRY else 0
     ) + (platform.system() == "Linux")  # xtl is not removed on Linux
     for p in res["actions"]["UNLINK"]:
-        assert p["name"] in env_pkgs
+        assert (
+            p["name"] in env_pkgs or p["name"] == "libstdcxx-ng"
+        )  # workaround special case lib not always removed
```

Comment:
`libstdcxx-ng` seems now to be used over `libstdcxx`, causing a test glitch.

This needs to be understood precisely.